wizard_builder/managers.py

```diff
@@ -12,22 +15,29 @@ def on_site(self, site_id=None):
         )
 
 
-class PageBaseManager(InheritanceManager):
+class FormQuestionQuerySet(InheritanceQuerySet):
+    pass
 
-    def get_queryset(self):
-        base_queryset = PageBaseQuerySet(self.model, using=self._db)
-        return base_queryset.select_subclasses()
 
-    def on_site(self, site_id=None):
-        return self.get_queryset().on_site(site_id)
+class AutoDowncastingManager(InheritanceManager):
 
+    def get_queryset(self):
+        base_queryset = self._queryset(self.model, using=self._db)
+        if getattr(settings, 'WIZARD_BUILDER_DISABLE_DOWNCASTING', False):
+            return base_queryset
+        # sourced from similar code in django-polymorphic
```

Comment:
out of an abundance of caution let's link the license ala https://github.com/SexualHealthInnovations/django-wizard-builder/blob/master/wizard_builder/views.py#L15-L20